docs/usermanual.md

```diff
@@ -240,6 +240,71 @@ attribute IDs `h` and `t`, then all measures (humidity and temperature) are repo
     $ mosquitto_pub -t /ABCDEF/id_sen1/attrs -m 'h|70|t|15' -h <mosquitto_broker> -p <mosquitto_port> -u <user> -P <password>
 ```
 
+##### Configuration retrieval
+
+The protocol offers a mechanism for the devices to retrieve its configuration (or any other value it needs from those
+stored in the Context Broker). Two topics are created in order to support this feature: a topic for configuration
+commands and a topic to receive configuration information.
+
+**Currenty, only the MQTT binding supports Configuration retrieval.**
+
+##### Configuration command topic
+
+```text
+/{{apikey}}/{{deviceid}}/configuration/commands
+```
+
+The IoT Agent listens in this topic for requests coming from the device. The messages must contain an Ultralight 2.0
+payload with the following format:
+
+```text
+{{type}}|{{fields}}
+```
+
+-   **type**: indicates the type of command the device is sending. See below for accepted values.
+-   **fields**: array with the names of the values to be retrieved from the Context Broker entity representing the
+    device, separated by the `|` character.
+
+This command will trigger a query to the CB that will, as a result, end up with a new message posted to the
+Configuration information topic (described bellow).
+
+E.g.:
+
+```text
+configuration|pollingInterval|publishInterval
+```
+
+There are two accepted values for the configuration command types:
+
+-   ~~`subscription`: this command will generate a subscription in the Context Broker that will be triggered whenever
```

Comment:
Mmm... it seems the ~~ remains :)

Hint: if your editor is causing problems maybe edit this line in github web itself would be a good idea.

Reply:
Ok, this is interesting. I have modified the line in Github editor and then copied to the .md document. It looks like ok in my browser.

![md](https://user-images.githubusercontent.com/6576853/60341921-5ae8a280-99b0-11e9-82d6-7a7de0fe79c3.png)

Reply:
Ah ok! You aim was to strike on the like...

Sorry the missunderstanding. NTC

Reply:
Just to emphasise that subscription is not supported 😄